SeaPublicWebsite/appsettings.DEV.json

```diff
@@ -11,5 +11,9 @@
 
     "GoogleAnalytics": {
         "MeasurementId": "G-RJC885GHSW"
+    },
+
+    "SimpleEnergyAdvice": {
+        "BaseUrl": "https://sea-beta-DEV.cloudapps.digital"

```

Comment:
Have yet to test whether is picked up correctly - will see if it works correctly when deploying to dev and whether puppeteer works then too.

Reply:
Are you adding the live url to prod as well?

Reply:
I made a similar change in `SeaPublicWebsite/appsettings.Production.json` but with the live URL if that's what you mean?